dashboards/rabbitmq/README.md

```diff
@@ -8,4 +8,13 @@
 |RabbitMQ Overview|
 |:------------------|
 |Filename: [rabbitmq-gce-overview.json](rabbitmq-gce-overview.json)|
-|This dashboard has charts displaying `Consumers`, `Current Messages`, `Messages Published`, `Messages Delivered`, `Acknowledged Messages`, `Dropped Messages`, `CPU % Top 5 VMs`, `Memory % Top 5 VMs` and `Hosts by Region`. There is also a card with links to docs and rabbitmq log in Cloud Logging.|
\ No newline at end of file
+|This dashboard has charts displaying `Consumers`, `Current Messages`, `Messages Published`, `Messages Delivered`, `Acknowledged Messages`, `Dropped Messages`, `CPU % Top 5 VMs`, `Memory % Top 5 VMs` and `Hosts by Region`. There is also a card with links to docs and rabbitmq log in Cloud Logging.|
+
+#### Notes
+
+- This dashboard is based on [built-in metrics](https://rabbitmq.com/prometheus.html).
+
+|RabbitMQ Overview|
```

Comment:
Can you add an entry for this dashboard to the metadata.yaml file https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/blob/master/dashboards/rabbitmq/metadata.yaml